Phenotypes.pm

```diff
@@ -258,8 +278,37 @@ sub run {
 
   return {} unless $data && scalar @$data;
 
+  return { PHENOTYPES =>  $data } if $self->{config}->{output_format} eq "json";
+
+  if ($DEFAULTS{phenotype_feature}){
+    my %result_uniq;
+    my @result_str = ();
+
+    foreach my $tmp_data(@{$data}) {
+      # get required data
+      my %tmp_return =
+        map {$_ => $tmp_data->{$_}}
+        grep {defined($cols{$_})}  # only include selected cols
+        keys %$tmp_data;
+
+      # replace link characters with _
+      $tmp_return{phenotype} =~ tr/ ;,)(/\_\_\_\_\_/;
+
+      # report only unique set of fields
+      my $record_line = join(",", values %tmp_return);
+      next if defined $result_uniq{$record_line};
+      $result_uniq{$record_line} = 1;
+
+      push(@result_str, join($char_sep, @tmp_return{@fields_order}));
+    }
+
+    return { PHENOTYPES => \@result_str }
```

Comment:
The `;` is missing at the end of the line

Reply:
Thanks, indeed!